conf/supervisord-dev.conf

```diff
@@ -3,7 +3,7 @@ nodaemon = true
 silent = true
 
 [program:web]
-command=gunicorn --paste conf/development.ini
+command=gunicorn --workers 1 --paste conf/development.ini
```

Comment:
Making the default explicit, this way is easier to change it in dev. 

I haven't found a good way to do it in the .ini file.